dist/tools/doccheck/check.sh

```diff
@@ -32,7 +32,7 @@ then
 fi
 
 exclude_filter() {
-    grep -v -e vendor -e examples -e tests
+    grep -v -e vendor -e examples -e tests -e "\<dist/tools\>"
```

Comment:
second question: why using `"\< \>"`. Note that it also works with just `dist/tools`

Reply:
to exclude corner cases like e.g. `absurdist/toolspinner` ;-) might also be a good idea to adapt this for the other pattrens if applicable. But that's a task for a different PR.

Reply:
ok